packages/back-end/src/models/SavedGroupModel.ts

```diff
@@ -0,0 +1,79 @@
+import mongoose from "mongoose";
+import { SavedGroupInterface } from "../../types/saved-group";
+import uniqid from "uniqid";
+
+const savedGroupSchema = new mongoose.Schema({
+  id: {
+    type: String,
+    unique: true,
+  },
+  orgId: {
+    type: String,
+    index: true,
+  },
+  groupName: String,
+  owner: String,
+  dateCreated: Date,
+  dateUpdated: Date,
+  values: [String],
+  attributeKey: String,
+});
+
+type SavedGroupDocument = mongoose.Document & SavedGroupInterface;
+
+const SavedGroupModel = mongoose.model<SavedGroupDocument>(
+  "savedGroup",
+  savedGroupSchema
+);
+
+function parseSaveGroupString(list: string) {
+  const listArr = list.split(",");
+
+  const savedGroup = listArr.map((i: string) => {
+    return i.trim();
+  });
+
+  return [
+    ...new Set(savedGroup.filter((value) => value !== "," && value !== "")),
+  ];
```

Comment:
The logic here looks fine.  Code style wise, I would have written this a little differently:

```ts
const values = list.split(",")
  .map((value) => value.trim())
  .filter((value) => !!value);

return [...new Set(values)]
```

Reply:
LOL - so much cleaner.